pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -70,8 +71,7 @@ public class MercuryOrderDao implements Dao<MercuryOrderDto> {
     public static String SQL_UPDATE_ORDER_STATUS =
             "UPDATE ddp_mercury_sequencing SET order_status = ?, status_date = ?, mercury_pdo_id = ?,  "
                     + " status_detail = ?, status_message = ? where order_id = ?";
-
-    private static final int MAX_CHAR_ALLOWED = 2000;
+    private static String SQL_GET_MERCURY_ORDER_ID = " SELECT * FROM ddp_mercury_sequencing WHERE order_id = ? ";
```

Comment:
How about `SQL_GET_MERCURY_ORDER_BY_ID`?  GET_MERCURY_ORDER_ID makes it sound like this is just going to return the order id.